### ISSUE TITLE
ES6 syntax error 

### ISSUE THREAD

**Issue by saadshahd** [bevry/ambi#5]
Using ambi with node v5.0.0/v0.10 popups a syntax error can you please explain ?  

```
(function (exports, require, module, __filename, __dirname) { export default function ambi (method, ...args) {
                                                              ^^^^^^

SyntaxError: Unexpected token export
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:404:25)
    at Object.Module._extensions..js (module.js:432:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:457:10)
    at startup (node.js:136:18)
    at node.js:972:3
```

**Comment by balupton** [comment]
Due to https://github.com/bevry/domain-browser/pull/7#issuecomment-160782405

Will fix over the week hopefully. Does it still work? As it should have a fallback for this.

Node 0.10 isn't supported by us anymore.

**Comment by saadshahd** [comment]
Thanks, yes it does.

**Comment by balupton** [comment]
Released v2.4.0 that should fix this. Thanks for reporting!
